src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,123 @@
+package duke.storage;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.exception.DukeNotTaskException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todo;
+
+
+/**
+ * A Class to handle file reading and writing.
+ *
+ * @author marioalvaro
+ */
+public class Storage {
+    private static final String PATH_FILE = "./data/TaskStorage.txt";
+    private File file = new File(PATH_FILE);
+
+    /**
+     * A method to read the txt file
+     * @return an ArrayList of Task
+     * @throws Exception of the scanner and DukeNotTaskException
+     */
+    public ArrayList<Task> readFile() throws Exception {
+        Scanner sc = new Scanner(file);
+        ArrayList<Task> readFile = new ArrayList<>();
+
+        while (sc.hasNextLine()) {
+            String scanned = sc.nextLine();
+            String[] singleTaskArray = scanned.split(" / ");
+
+            //do a checking <------
```

Comment:
I'm not entirely sure what this comment means. Perhaps just a `//TODO` for yourself? I understand that a lot of this is still WIP, for example I noticed some kind of template above like "Replace this stub with your completed method.". But just thought I'd mention this anyway.

![image](https://github.com/nus-cs2103-AY2324S1/ip/assets/44526554/c2a0aa46-4da0-4b73-8eb9-ec269a27e908)